### PR TITLE
I2B2UI-292: Hide "Load More" link when all QM are loaded in the CRC History window

### PR DESCRIPTION
**Pull request by hackerceo**

### CODE REVIEW

**Comment by marcdanie** [comment]
I am still seeing the Load More link when there are zero previous queries. Is that the expected behavior?

**Comment by hackerceo** [comment]
Fixed a typo, the bug occurring on initial load should be fixed now

File: js-i2b2/cells/CRC/CRC_view_History.js [hunk 1]
```diff
@@ -70,34 +70,133 @@ i2b2.CRC.view.history.loadChildren = function(ev, nodeData) {
...
 i2b2.CRC.view.history.loadMore = function() {
     $('.history-more-bar i.bi').removeClass("d-none");
     let newcount = i2b2.CRC.view.history.treeview.data('treeview').getNodes(()=>true).length;
     newcount = newcount + i2b2.CRC.view.history.params.maxQueriesDisp;
     i2b2.CRC.view.history.LoadQueryMasters(newcount);
 }
 
+// ================================================================================================== //
+i2b2.CRC.view.history.clickSearchName = function() {
+    console.info("CALLED i2b2.CRC.ctrlr.history.clickSearchName()");
```
Comment (marcdanie) [inline js-i2b2/cells/CRC/CRC_view_History.js:89]: Minor typo in name of function

**Review by marcdanie — APPROVED** [review]
Small typo but looks good.

**Pull request merged**